src/pages/TransactionsPage/TransactionsForm/TransactionsForm.js

```diff
@@ -0,0 +1,178 @@
+import { StyledTransactionForm } from "./TransactionsForm.styled";
+import { DatePicker } from "../../../components/form/DatePicker/DatePicker";
+import { Input } from "../../../components/form/Input/Input";
+import { RadioButton } from "../../../components/form/RadioButton/RadioButton";
+import { Dropdown } from "../../../components/form/Dropdown/Dropdown";
+import { useInputState } from "../../../services/hooks/useInputState";
+import { useState, useEffect } from "react";
+import { validate } from "../../../services/helpers/formValidationRules";
+import Modal from "../../../components/Modal";
+import { useModal } from "../../../services/hooks/useModal";
+
+export const TransactionForm = () => {
+  const { isVisible, toggleVisibility } = useModal();
+
+  const [title, setTitle, resetTitle] = useInputState("");
+  const [description, setDescription, resetDescription] = useInputState("");
+  const [amount, setAmount, resetAmount] = useInputState(0);
+  const [type, setType, resetType] = useInputState("income");
```

Comment:
do not use hardcode values, consider creating some kind of enum for that

Reply:
Can you please clarify what you meant by enum? Should I just create separate variables and pass it to state or... ? Not sure what you mean.

Reply:
it might be something like 
```
const transactionFormFields = {
     income: 'income',
     expense: 'expense',
     ...
}
```

`useInputState(transactionFormFields.expense)`